### PR TITLE
Login redirect

### PR DESCRIPTION
**Pull request by kel29**

### CODE REVIEW

File: src/containers/Login.js [hunk 1]
```diff
@@ -14,7 +15,8 @@ class Login extends PureComponent {
 
   userLogin = (e) => {
     e.preventDefault()
-    let currentUser = this.props.users.find(user => user.email.toLowerCase() === this.state.email.toLowerCase() && user.name.toLowerCase() === this.state.username.toLowerCase())
+    console.log(this.state.email.toLowerCase(), this.state.username.toLowerCase())
```
Comment (renburnett) [inline src/containers/Login.js]: oop forgot to remove console.log

**Review by renburnett — COMMENTED** [review]
sweet!

File: src/containers/Login.js (hunk 1 above)
Comment (kel29) [reply src/containers/Login.js]: thanks! updated

**Review by renburnett — APPROVED** [review]

**Review by ianhunterharold — APPROVED** [review]
all looks good.

**Pull request merged**